### ISSUE TITLE
Consider removing the "@bors-servo" requirement from homu commands?

### ISSUE THREAD

**Issue by larsbergstrom** [servo/saltfs#624]
IIRC, homu supports not requiring the @bors-servo bit. Should we consider removing it? I just went through my third conversation in the past two weeks explaining the history of the `bors` name and I'm pretty sure it's not serving a huge purpose above & beyond just the usual `r+` or other commands on a single line by themselves.

cc @edunham @metajack @Manishearth

**Comment by KiChjang** [comment]
Wouldn't this cause confusion with the "r=me" comments?

**Comment by larsbergstrom** [comment]
Yeah, I guess we'd have to ask people to do `delegate+` instead, assuming that `r=me` is "you can `r+` this when you address your nits"?

**Comment by SimonSapin** [comment]
Or teach homu that `me` is not an acceptable reviewer in commands?

**Comment by metajack** [comment]
Or require the command be on a line alone?

**Comment by Manishearth** [comment]
IIRC homu doesn't support it. Homu does support @homu though.

We probably could add support for nameless pings.

**Comment by aneeshusa** [comment]
I kind of like being explicit about giving commands to Homu, e.g. to allow explaining delegation without triggering a merge. Our [wiki page on Homu](https://github.com/servo/servo/wiki/Homu) mentions the name history - do you think we could just point people at that (and maybe add a bit more of the history)?

**Comment by Manishearth** [comment]
Works for me. Also, we can add aliases like @homu or @bors

**Comment by metajack** [comment]
Won't those cause spurious notifications to the owners of those accounts?

**Comment by Manishearth** [comment]
both are already bot accounts, presumably with notifications memoryholed. They both run homu, which does not listen to notifications; rather it looks for pings in the full text of comments on repos it is configured to listened to.

**Comment by larsbergstrom** [comment]
Like that the suggestion for using @homu, since then it's at least one consistent name :-)

@aneeshusa It's more that as we start having Gecko/Firefox people have to interact with this part of the system, it'd be nice to remove a bit of the legacy stuff. I find that even people who know the history end up calling homu "bors" from time to time (or use `bors` as a verb), further complicating it. I'm also one of them, as seen from "larsbors" and the recent "glennbors" as our moniker for human-carry when the bots go AWOL.

**Comment by metajack** [comment]
Homu is the software, not the name of the bot. Have we be leaking this implementation detail? A better bot name would be nice.

**Comment by Manishearth** [comment]
We leak it in discussions about the software.  Recently someone accidentally looked at the bors repo instead of homu, I forget who.

Note that the invocation doesn't even need to ping someone, it can be `homu: r+` or something. bors-ng does this.

**Comment by edunham** [comment]
I think that it's intuitive to address the autoland-bot by name (with or without the @) when giving it a command. Upon actually thinking about it, it seems really odd to request that @bors-servo take some action by saying "@homu: do the thing"

Any bot with Homu or Bors in its name will have the same namespace-overloading issues. Heck, the original "Bors" was that thingy Graydon wrote that ran on a cron job and Homu was written to replace. It seems like the only way out of this confusion is to bring in an entirely new name to address "whichever bot happens to be autolanding all our code right now". 

The bad part about a new name is that  we would have to retrain active contributors used to talking to bors-es and homus. The good part is that a differently-named account with sufficient perms could edit peoples' github comments to correct its name whenever it was referred to by an old alias, thus minimizing confusion.
